### PR TITLE
add skip button to hero page

### PR DESCRIPTION
**Pull request by malhotra5**
**End-user friendly description of the problem this fixes or functionality that this introduces**

- [ ] Include this change in the Release Notes. If checked, you must provide an **end-user friendly** description for your change below

---
**Give a summary of what the PR does, explaining any non-trivial design decisions**

The linked issue was going stale and seemed quite helpful to have. Added a "skip" button to bypass the hero page. Followed general design theme for the button but feedback here would be appreciated!

---
**Link of any specific issues this addresses**
#4503

### CODE REVIEW

File: frontend/src/routes/_oh._index/route.tsx [hunk 1]
```diff
@@ -140,6 +140,15 @@ function Home() {
           />
         </div>
       </div>
+      <button
+        className="absolute bottom-4 right-4 text-white px-4 py-2 rounded cursor-pointer border-1 border-neutral-600"
+        onClick={() => {
+          window.location.href = "/app";
```
Comment (amanape) [inline frontend/src/routes/_oh._index/route.tsx:146]:
```suggestion
          navigate("/app");
```
Further reading: https://remix.run/docs/hi/main/hooks/use-navigate

**Review by amanape — COMMENTED** [review]
I’m not entirely sure if the original issue has been resolved. When a user uploads a project directory, they’re already redirected immediately to `/app`. @tobitege, could you confirm if this was the intended behavior?  

In any case, we’ll need confirmation from Paul, the designer. "Skip" feels out of place here (there isn't quite anything to skip). Something like "Jump right in" might be more fitting.

**Comment by tobitege** [comment]
@amanape Personally, I don't plan to use this upload feature, but still use mapped local folder as my codebase doesn't use GitHub.
Whichever option would allow to skip the hero page would be fine with me. 😃

**Comment by mamoodi** [comment]
@amanape was this discussed with Paul?

**Comment by amanape** [comment]
@mamoodi Nope, Paul is occupied as of yet

**Comment by malhotra5** [comment]
Closing this PR for now as there's no clear next steps. Also I believe navigation is done [here](https://github.com/All-Hands-AI/OpenHands/blob/main/frontend/src/routes/_oh._index/route.tsx#L37) when token present

**Pull request closed without merging**